relayer/relayMsgs.go

```diff
@@ -45,8 +45,10 @@ func (r *RelayMsgs) IsMaxTx(msgLen, txSize uint64) bool {
 // Send sends the messages with appropriate output
 // TODO: Parallelize? Maybe?
 func (r *RelayMsgs) Send(src, dst *Chain) {
-	var msgLen, txSize uint64
-	var msgs []sdk.Msg
+	var (
+		msgLen, txSize uint64
+		msgs           = make([]sdk.Msg, len(r.Src))
```

Comment:
I think this should probably not pre alloc. msgs is used to batch send msgs that need to be relayed by using preset max tx length (in bytes) or send a max amount of msgs using msgLen, unfortunately msgLen is not always set. Also, it is used by r.Dst which may not have the same length as r.Src